gfw_pixetl/tiles/vector_src_tile.py

```diff
@@ -135,23 +137,19 @@ def fetch_data(self) -> None:
         engine = create_engine(db_url)
 
         val_column = literal_column(str(self.layer.calc))
+        # geom_column = literal_column(GEOMETRY_COLUMN)
```

Comment:
Maybe remove this line, unless you want to add a comment on why this is a possibility for replacing line 141, but you didn't do it right now.

Reply:
Done, thanks!